### PR TITLE
Add no-qa instructions

### PR DESCRIPTION
**Pull request by langalex**
* what this PR does: add instructions on when and how no to do QA
* why this is needed: sometimes it makes no sense to QA a PR
* context for anyone reading this in 6 months or years: see https://www.notion.so/cobotme/define-a-no-QA-label-process-43d842078e3040a0bd4fdf5df3f1f80a

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -5,19 +5,32 @@
...
+# QA
+
+After review is done.
+
+If no QA is needed, add "No QA" label before requesting a review.
+
+<details>
+  <summary>No QA criteria</summary>
+  <ul>
+    <li>API endpoints</li>
```
Comment (SubandiK) [inline .github/pull_request_template.md]: I think it has at least once happened that during QA of an endpoint PR, the QA person found something that lead to a change, so the value of those is > 0.

The QA 'style' there was not "checking if things look good" and more "can I break this?". Or should this already be covered during the review step by the reviewer asking questions and the dev being responsible to check them?

**Review by SubandiK — APPROVED** [review]
approved for technicality, added a comment / question about if "api endpoint should always mean no QA"

File: .github/pull_request_template.md (hunk 1 above)
Comment (langalex) [reply .github/pull_request_template.md]: yeah good point. i find it hard to come up with a waterproof criterion for when (not) to QA api endpoints. any ideas?

Comment (SubandiK) [reply .github/pull_request_template.md]: hm, I also find it hard to come up with a "rule" that isn't too generic or specific.
One good reason to QA would be if there are many different scenarios where the endpoint is already in use, i.e. if there are already "clicky" ways to QA?

Comment (langalex) [reply .github/pull_request_template.md]: changed it a bit. people need to decide for themselves.

**Pull request merged**